source/portable/NetworkInterface/WinPCap/NetworkInterface.c

```diff
@@ -755,7 +755,7 @@ static BaseType_t xPacketBouncedBack( const uint8_t * pucBuffer )
 }
 /*-----------------------------------------------------------*/
 
-extern NetworkEndPoint_t* pxGetEndpoint( BaseType_t xIPType );
```

Comment:
Strange that WinPCap/NetworkInterface.c shows many changes while it is equal to the dev/integration branch.

Reply:
Made the fixes, currently in sync.